apps/backend/core/configuration/validation/base.py

```
@@ -0,0 +1,279 @@
+
+            async def validate(self) -> ValidationResult:
+                # Validation logic here
+                pass
+        ```
+    """
+
+    def __init__(
+        self,
+        service_name: str,
+        level: ValidationLevel,
+        config: Any,
```

Ruff (0.14.4)

Dynamically typed expressions (typing.Any) are disallowed in `config`

(ANN401)